packages/bitcore-lib/test/publickey.js

```diff
@@ -451,4 +451,36 @@ describe('PublicKey', function() {
 
   });
 
+  describe('#isValidTaproot', function() {
+    it('should be true - hex string', function() {
+      const isValid = PublicKey.isValidTaproot('29b21ed3959615c97d0866a295486c25577aaa5ee18d463489d9dbe3cf0aaf5e');
+      isValid.should.equal(true);
+    });
+
+    it('should be true - buffer', function() {
+      const isValid = PublicKey.isValidTaproot(Buffer.from('29b21ed3959615c97d0866a295486c25577aaa5ee18d463489d9dbe3cf0aaf5e', 'hex'));
+      isValid.should.equal(true);
+    });
+
+    it('should be false - hex string', function() {  
```

Comment:
If you view tests as documentation, then either a more descriptive test name, or even just a comment in the test, would be useful. Why is this input invalid when the above input is valid?

Reply:
invalid X